lib/graphql/list_type.rb

```diff
@@ -17,8 +17,13 @@ def to_s
     "[#{of_type.to_s}]"
   end
 
-  def coerce_input(value)
-    inner_type = of_type
-    value.map { |item| inner_type.coerce_input!(item) }
+  def valid_non_null_input?(value)
+    value = [value] unless value.is_a?(Array)
```

Comment:
What's the case where a list type is asked to validate a single item? Should it always be given an array?